tools/Microsoft.WindowsAzure.Build.Tasks/FilterOutAutoRestLibraries.cs

```diff
@@ -43,6 +52,18 @@ public override bool Execute()
             var netCoreAutoRestLibraries = new List<ITaskItem>();
             var netCoreLibraryTestOnes = new List<ITaskItem>();
             var others =  new List<ITaskItem>();
+            
+            List<string> nPkgsList = null;
+
+            if (NugetPackagesToPublish != null)
+            {
+                NugetPackagesToPublish = NugetPackagesToPublish.Trim();
+
+                if (!string.IsNullOrEmpty(NugetPackagesToPublish))
+                {   
+                    nPkgsList = NugetPackagesToPublish.Split(' ').ToList<string>();
+                }
```

Comment:
you should Split using IgnoreWhiteSpaces parameter. if there will be 2 spaces or more in the passed parameter this part may not work correct